yarn-project/foundation/src/abi/abi.ts

```diff
@@ -390,7 +390,14 @@ export function getFunctionArtifact(
   if (!functionArtifact) {
     throw new Error(`Unknown function ${functionNameOrSelector}`);
   }
-  const debugMetadata = getFunctionDebugMetadata(artifact, functionArtifact);
+  // TODO(#10546) investigate why debugMetadata was so big, for some tests https://github.com/AztecProtocol/aztec-packages/issues/10546, and whether it even needs to be retrieved at all, here.
+  let debugMetadata = undefined;
+  try {
+    debugMetadata = getFunctionDebugMetadata(artifact, functionArtifact);
```

Comment:
For some reason, the debug_symbols field of one of the functions in the `StatefulTest` contract became _huge_, and when the tests tried to grab that function's artifact, it wouldn't be able to parse it (too big). See the `inflate` function being called in the function directly below this one.
See https://github.com/AztecProtocol/aztec-packages/issues/10546
Anyway, this bodge fix only assigns debugMetadata if it can be parsed.